src/document.c

```diff
@@ -472,6 +474,8 @@ FIELD_BULK_INDEXER(geoIndexer) {
     QueryError_SetError(status, QUERY_EGENERIC, "Could not index geo value");
     return -1;
   }
+  ctx->spec->stats.invertedSize += GEO_STORAGE_SIZE; // TODO: exact amount
+  ctx->spec->stats.numRecords++;
```

Comment:
Maybe we should not count the geo because it's not really an inverted index that we keep and redis counts it in its global memory report?

Reply:
@gkorland @mnunberg 
Should we count GEO here?

> Why not just update the stats on InvertedIndex_WriteEntryGeneric instead of returning the size all the way back?

The stats are recorded on the spec which isn't available there.